extensions/eda/plugins/event_source/file_watch.py

```diff
@@ -35,7 +34,10 @@ def watch(
     root_path = args["path"]
 
     class Handler(RegexMatchingEventHandler):
-        def __init__(self: "Handler", **kwargs: Optional(list[str])) -> None:
+        def __init__(
+            self: "Handler",
+            **kwargs: None | list[str],  # pylint: disable=E1131
```

Comment:
Regardless of the type hint issue, the type hint it self is wrong. Unpacking keywords arguments creates a dict. 
It should be:
```suggestion
            **kwargs: None | dict,  # pylint: disable=E1131
```

Pylint should not raise E1131 here, it might be a bug. 

Anyway, this new syntax is implemented in python 3.10, but we support from python3.9, so the final code should be:
```suggestion
            def __init__(self: "Handler", **kwargs: dict) -> None
```